pymodbus/client/asynchronous/__init__.py

```diff
@@ -31,4 +31,18 @@
     # For asyncio the actual client is returned and event loop is asyncio loop
 
 """
+<<<<<<< HEAD:pymodbus/client/asynchronous/__init__.py
 from pymodbus.client.asynchronous.deprecated.asynchronous import *
+=======
+from pymodbus.compat import is_installed
+
+installed = is_installed('twisted')
+if installed:
+    # Import deprecated async client only if twisted is installed #338
+    from pymodbus.client.async.deprecated.async import *
+else:
+    import logging
+    logger = logging.getLogger(__name__)
+    logger.warning("Not Importing deprecated clients. "
+                   "Dependency Twisted is not Installed")
+>>>>>>> 7980f087a0ecbf24ca049849799b365892082029:pymodbus/client/async/__init__.py
```

Comment:
Just a heads up, you managed to commit the merge conflict here. :wink:

Reply:
Shame on me for doing the merge so late during the night. I did it in  the wrong branch too :)

This one will be closed the new one on the correct branch has been sent